app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/TicketType.kt

```diff
@@ -13,7 +13,7 @@ import com.google.gson.annotations.SerializedName
  */
 @Entity
 data class TicketType(@PrimaryKey
-                  @SerializedName("tickettype")
+                  @SerializedName("ticket_type")
                   var id: Int = 0,
```

Comment:
Why not call this in the name that actually is used everywhere else?

Reply:
I think with "id" it is more intuitive what this is, and that it's just an Integer.

Reply:
@schulzalexander in `TicketReservation` you have it named differently. Please just make it consistent.

Reply:
also named it "id" in TicketReservation.